core/XH.js

```diff
@@ -467,7 +467,7 @@ class XHClass {
             try {
                 await this.installServicesAsync(EnvironmentService);
             } catch (e) {
-                throw `Unable to load environment info - is the server running and reachable? (${e.message})`;
+                throw `UI Server is unavailable - please contact support. (${e.message})`;
```

Comment:
Any reason we aren't using XH.exception() here?

Reply:
We could do something like XH.exception({
message: 'UI Server is unavailable - please contact support.'
detail: Some details of exception including the actual *URL*
})

Would ideally allow users to copy all details to clipboard, if not mail them  (although not sure we have easy way to do one without the other, and recall, mailing unlikely to work without backend)

Reply:
Did this. User was always, and still is, able to copy the details to the clipboard, but yes would have to email with another service. No backend to save to, and cannot set up email with only JS.

![Screen Shot 2019-07-05 at 9 15 43 AM](https://user-images.githubusercontent.com/15745614/60725211-7bf84880-9f06-11e9-884f-05b13a51e44c.png)

Reply:
The latest:

![Screen Shot 2019-07-05 at 10 01 00 AM](https://user-images.githubusercontent.com/15745614/60727424-dba52280-9f0b-11e9-8808-62dbb7e32223.png)